script.js

```diff
@@ -0,0 +1,30 @@
+//Declare and initilize variables
+let howOldAreYou;
+let invalidInput;
+let Age;
+let approvedAge = 21;
+const WELCOME_MESSAGE = `Welcome To The Kosher Wine Cellar!
+You must be over 21 years to purchase on our site!`;
+
+alert(WELCOME_MESSAGE);
+
+Age = prompt('Please Enter Your Age');
```

Comment:
Instead of splitting the declaration and assignment, simply declare age here
```suggestion
let age = prompt('Please Enter your Age');
```